### ISSUE TITLE
Makes an example for Font Awesome 6 and searchPseudoElements

### ISSUE THREAD

**Issue by jmverges** [FortAwesome/Font-Awesome#18670]
**Describe the bug**
I'm trying without luck to transform to svg a span with a custom class "pi"

```
    <span class="fa fa-solid fa-award"></span>
    <span class="pi pi-solid pi-award"></span>
```
I see how <span class="fa fa-solid fa-award"> gets translated to SVG but <span class="pi pi-solid pi-award"></span> is not
I have defined ```$fa-css-prefix: "pi";``` and I can see the icon but using webfont instead of svg

I tried this example https://codepen.io/fontawesome/pen/WVEobv but it doesn't work with Font Awesome 6

**To Reproduce**
You can copy what it is in https://codepen.io/fontawesome/pen/WVEobv and just change the font name it doens't work

**Expected behavior**
Have an example for Font Awesome 6 Pro

- [x] SVG with JS
- [x] Web Fonts with CSS
- [ ] SVG Sprites
- [ ] On the Desktop

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [x] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and apologies for the late reply.

> I tried this example https://codepen.io/fontawesome/pen/WVEobv but it doesn't work with Font Awesome 6

This example does not work with Font Awesome 6 because:
1. the `font-family` css rule is not correct
2. codepen.io is not (yet?) allowlisted to use FA6 Pro

> I have defined $fa-css-prefix: "pi"; and I can see the icon but using webfont instead of svg

This should not work with the SVG+JS Framework. You may be interested in the `familyPrefix` option: https://fontawesome.com/docs/apis/javascript/configuration#familyprefix, but not for this use case

**Comment by robmadole** [comment]
@jmverges I've updated https://codepen.io/fontawesome/pen/WVEobv to use v6 with a Kit. Does that help?

**Comment by jmverges** [comment]
thanks @robmadole 
is not bad, however we use in webpack so no possibility to add data-search-pseudo-elements. 
On the other hand, I got it working but the performance was not really good when dom.watch is present so we are instead using just the font instead svg.
We needed to use dom.watch because we wanted to homogenize some icons inside another 3rd party js components lib which we cannot hook in a different way.
I'm going to close thise one. 
Thanks!
